netbox_agent/network.py

```diff
@@ -80,14 +81,18 @@ def get_netbox_type_for_nic(self, nic):
 
     def create_netbox_nic(self, device, nic):
         # TODO: add Optic Vendor, PN and Serial
+        type = self.get_netbox_type_for_nic(nic)
+        logger.info('Creating NIC {name} ({mac}) on {device}'.format(
+            name=nic['name'], mac=nic['mac'], device=device.name))
         return nb.dcim.interfaces.create(
             device=device.id,
             name=nic['name'],
             mac_address=nic['mac'],
-            type=self.get_netbox_type_for_nic(nic),
+            type=type,
         )
 
     def create_netbox_network_cards(self):
+        logger.info('Creating NIC..')
```

Comment:
```suggestion
        logger.info('Creating NIC...')
```

Also, I feel this should be of DEBUG level; you should only keep the most vital information as INFO.